models/product/src/product/product-draft/presets/sample-data-fashion/anniversary-shirt.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  KeyReference,
+  LocalizedString,
+} from '@commercetools-test-data/commons';
+import { ProductVariantDraft } from '@commercetools-test-data/product-variant';
+import * as ProductDraft from '../../../product-draft';
+import { TProductDraftBuilder } from '../../../types';
+
+const anniversaryShirt = (): TProductDraftBuilder =>
+  ProductDraft.presets
+    .empty()
+    .name(LocalizedString.presets.empty()['en-US']('Sample Anniversary Shirt'))
+    .slug(LocalizedString.presets.empty()['en-US']('sample-anniversary-shirt'))
+    .productType(KeyReference.presets.productType().key('shirts'))
```

Comment:
The key here (and in the rest of the product drafts) should be pulled in from the shirt product type preset

Reply:
Completely spaced that - fixing!